src/work-stealing-queue.h

```diff
@@ -35,15 +35,15 @@ static inline ws_array_t *create_ws_array(size_t capacity, int32_t eltsz) JL_NOT
 }
 
 typedef struct {
-    _Atomic(int64_t) top;
-    _Atomic(int64_t) bottom;
```

Comment:
Are top and bottom reset after marking?

Reply:
No

Reply:
It's a circular buffer, so we don't really need to

Reply:
ah so this is correct under overflow? Do we then not need to make them uint32_t?

Reply:
Ah OK. Le et al.'s paper uses a `size_t`, so better play it safe.